index.html

```diff
@@ -1,91 +1,143 @@
+            <div class="big-text">
+              구매를 원하는 <br />
+              상품을 검색하세요
             </div>
-        </section>
-        <section class="bottom-section">
-            <div class="bottom-section-container">
-                <div class="bottom-section-text">
-                    <p>믿을 수 있는 <br> 판다마켓 중고 거래</p>
-                </div>
-                <img src="img/home/panda-image-bottom.png" alt="귀여운 판다 두마리가 중고 거래하는 이미지">
+            <p>
+              구매하고 싶은 물품은 검색해서 <br />
+              쉽게 찾아보세요
+            </p>
+          </div>
+          <img
+            src="img/home/main-img-2.png"
+            alt="원하는 상품을 검색하는 이미지"
+            class="search-img"
+          />
+        </div>
+      </section>
+      <section class="main-section">
+        <div class="register">
+          <img
+            src="img/home/main-img-3.png"
+            alt="연필, 필통, 도서, 안경 같은 물건을 등록하는 이미지"
+            class="register-img"
+          />
+          <div class="text-container">
+            <div class="small-text">Register</div>
+            <div class="big-text">
+              판매를 원하는 <br />
+              상품을 등록하세요
             </div>
-        </section>
+            <p>
+              어떤 물건이든 판매하고 싶은 상품을 <br />
+              쉽게 등록하세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="bottom-section">
+        <div class="bottom-section-container">
+          <div class="bottom-section-text">
+            <p>
+              믿을 수 있는 <br />
+              판다마켓 중고 거래
+            </p>
+          </div>
+          <img
+            src="img/home/panda-image-bottom.png"
+            alt="귀여운 판다 두마리가 중고 거래하는 이미지"
+          />
+        </div>
+      </section>
     </main>
 
     <footer>
-        <div class="codeit">©codeit - 2024</div>
-        <div class="footer-center">
-            <a href="/privacy.html" class="privacy-policy"> Privacy Policy </a>
-            <a href="/faq.html" class="faq"> FAQ </a>
-        </div>
-        <div class="footer-right">
-            <a href="https://www.facebook.com" target="_blank"><img src="img/home/facebook-icon.png" alt="페이스북 로고"></a>
-            <a href="https://www.twitter.com" target="_blank"><img src="img/home/twitter-icon.png" alt="트위터 로고"></a>
-            <a href="https://www.youtube.com" target="_blank"><img src="img/home/youtube-icon.png" alt="유튜브 로고"></a>
-            <a href="https://www.instagram.com" target="_blank"><img src="img/home/instagram-icon.png"
-                    alt="인스타그램 로고"></a>
-        </div>
+      <div class="codeit">©codeit - 2024</div>
+      <div class="footer-center">
+        <a href="/privacy.html" class="privacy-policy"> Privacy Policy </a>
+        <a href="/faq.html" class="faq"> FAQ </a>
+      </div>
+      <div class="footer-right">
+        <a href="https://www.facebook.com" target="_blank"
+          ><img src="img/home/facebook-icon.png" alt="페이스북 로고"
+        /></a>
```

Comment:
💊 제안
현재 코드에서는 하나의 태그가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~


```suggestion
        <a href="https://www.facebook.com" target="_blank">
         <img src="img/home/facebook-icon.png" alt="페이스북 로고"/>
        </a>
```